libnetwork/netutils/utils_linux_test.go

```diff
@@ -186,22 +189,28 @@ func TestNetworkRange(t *testing.T) {
 
 // Test veth name generation "veth"+rand (e.g.veth0f60e2c)
 func TestGenerateRandomName(t *testing.T) {
-	name1, err := GenerateRandomName("veth", 7)
-	if err != nil {
-		t.Fatal(err)
-	}
-	// veth plus generated append equals a len of 11
-	if len(name1) != 11 {
-		t.Fatalf("Expected 11 characters, instead received %d characters", len(name1))
-	}
-	name2, err := GenerateRandomName("veth", 7)
-	if err != nil {
-		t.Fatal(err)
-	}
-	// Fail if the random generated names equal one another
-	if name1 == name2 {
-		t.Fatalf("Expected differing values but received %s and %s", name1, name2)
-	}
+	const vethPrefix = "veth"
+	const vethLen = 11
+
+	name1, err := GenerateRandomName(vethPrefix, vethLen)
+	assert.NilError(t, err)
+	assert.Assert(t, strings.HasPrefix(name1, vethPrefix), "Expected name to start with %s", vethPrefix)
+	assert.Equal(t, len(name1), vethLen, "Expected %d characters, instead received %d characters", vethLen, len(name1))
```

Comment:
```suggestion
	assert.Check(t, is.Len(name1, vethLen))
```
(Moby convention is to import [`"gotest.tools/v3/assert/cmp"`](https://pkg.go.dev/gotest.tools/v3@v3.4.0/assert/cmp) as `is`.)